README.md

```diff
@@ -78,6 +83,11 @@ A template of the `licenses.json` file can be found in the repository at `src/ma
     java -jar lpvs-1.0.0.jar
     ```
 
+   When running the application you will also be able to use command line to input all the same values associated with github and license on the fly, like so:
+   ```bash
+   java -jar lpvs-1.0.0.jar -Dgithub.login=<`my-login`> -Dgithub.token=<`my-token`> etc.
```

Comment:
@v-kerimov  Incorrect command.
-D flag should be added before JAR file name

java -jar  -Dgithub.login=<`my-login`> -Dgithub.token=<`my-token`> etc.  lpvs-1.0.0.jar